build/params_calibnet.go

```diff
@@ -73,11 +73,11 @@ const UpgradeSharkHeight = 16800 // 6 days after genesis
 // 2023-02-21T16:30:00Z
 const UpgradeHyggeHeight = 322354
 
-// ??????????
-const UpgradeLightningHeight = 999999999999999
+// 2023-04-18T16:30:00Z
```

Comment:
Someone please check this, I just added 7*8*2880 since this is exactly 8 weeks after the last upgrade, I might be off here.

Reply:
This is correct, yes.